pkg/cluster_upgrader_builder/cluster_upgrader_builder.go

```diff
@@ -1,22 +1,22 @@
 package cluster_upgrader_builder
```

Comment:
A good opportunity to correct the package name?
https://blog.golang.org/package-names#TOC_2.

Reply:
In commit 54e35d9 I have refactored the `cluster_upgrader_builder` into the `upgraders` package (it doesn't really gain anything from sitting outside there) and renamed it to `builder.go`.